test/loopback-dl.test.js

```diff
@@ -1120,7 +1120,51 @@ describe('Model define with relations configuration', function() {
       keyTo: 'id',
       multiple: false,
       polymorphic: {
-        as: 'imageable',
+        reference: 'imageable',
+        foreignKey: 'imageableId',
+        discriminator: 'imageableType',
+      },
+    });
+    done();
+  });
+
+  it('sets up hasMany polymorphic relation with `{polymorphic: belongsToRelationName}`', function(done) {
+    var ds = new DataSource('memory');
+
+    var Picture = ds.define('Picture', {name: String}, {relations: {
+      imageable: {type: 'belongsTo', polymorphic: true},
```

Comment:
@ebarault hmm, sorry I don't get what is the difference between this test 
> 'sets up hasMany polymorphic relation with `{polymorphic: belongsToRelationName}`' 

and 

> 'sets up belongsTo polymorphic relation with `{polymorphic: true}`'

Are you testing the model setup sequence? I think we already cover that in https://github.com/strongloop/loopback-datasource-juggler/pull/1301/files#diff-e7b7d30bbb20e62309db0af3bd046c71R2251